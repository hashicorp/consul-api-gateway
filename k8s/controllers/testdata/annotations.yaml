
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: Gateway
metadata:
  name: annotations
  annotations:
    "polar.hashicorp.com/auth-method": "polar"
    "polar.hashicorp.com/image": "hashicorp/polar:0.0.0"
    "polar.hashicorp.com/service-account": "polar"
    "polar.hashicorp.com/envoy": "hashicorp/special-envoy:0.0.0"
    "polar.hashicorp.com/log-level": "debug"
    "polar.hashicorp.com/consul-http-address": "consul.localhost"
    "polar.hashicorp.com/consul-http-scheme": "http"
spec:
  listeners:  
  - protocol: HTTP
    port: 80
    name: http
    allowedRoutes:
      namespaces:
        from: Same