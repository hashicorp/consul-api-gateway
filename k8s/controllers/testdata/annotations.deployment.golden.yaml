metadata:
  creationTimestamp: null
  labels:
    managedBy: polar
    name: polar-annotations
  name: annotations
spec:
  selector:
    matchLabels:
      managedBy: polar
      name: polar-annotations
  strategy: {}
  template:
    metadata:
      creationTimestamp: null
      labels:
        managedBy: polar
        name: polar-annotations
    spec:
      containers:
      - command:
        - /bootstrap/polar
        - exec
        - -log-json
        - -log-level
        - debug
        - -gateway-host
        - $(IP)
        - -gateway-ports
        - HTTP:http:80
        - -gateway-name
        - annotations
        - -consul-http-address
        - consul.localhost
        - -consul-http-port
        - "8500"
        - -consul-xds-port
        - "8502"
        - -envoy-bootstrap-path
        - /bootstrap/envoy.json
        - -envoy-sds-address
        - polar-controller.default.svc.cluster.local
        - -envoy-sds-port
        - "9090"
        - -acl-auth-method
        - polar
        env:
        - name: IP
          valueFrom:
            fieldRef:
              fieldPath: status.podIP
        - name: HOST_IP
          valueFrom:
            fieldRef:
              fieldPath: status.hostIP
        image: hashicorp/special-envoy:0.0.0
        name: polar
        ports:
        - containerPort: 80
          name: http
          protocol: TCP
        resources: {}
        volumeMounts:
        - mountPath: /bootstrap
          name: bootstrap
        - mountPath: /certs
          name: certs
      initContainers:
      - command:
        - cp
        - /bin/polar
        - /bootstrap/polar
        image: hashicorp/polar:0.0.0
        name: polar-init
        resources: {}
        volumeMounts:
        - mountPath: /bootstrap
          name: bootstrap
        - mountPath: /certs
          name: certs
      serviceAccountName: polar
      volumes:
      - emptyDir: {}
        name: bootstrap
      - emptyDir: {}
        name: certs
status: {}
